src/libraries/System.Private.CoreLib/src/System/Numerics/Quaternion.cs

```diff
@@ -6,66 +6,38 @@
 
 namespace System.Numerics
 {
-    /// <summary>
-    /// A structure encapsulating a four-dimensional vector (x,y,z,w),
-    /// which is used to efficiently rotate an object about the (x,y,z) vector by the angle theta, where w = cos(theta/2).
-    /// </summary>
+    /// <summary>A structure encapsulating a four-dimensional vector (x,y,z,w), which is used to efficiently rotate an object about the (x,y,z) vector by the angle theta, where w = cos(theta/2).</summary>
```

Comment:
I'm not sure this is more readable since now the reader has to horizontally scroll to read the whole thing.

Reply:
We don't consistently cap line length anywhere else and these docs are hardly used. I don't even think they match what shows in current IntelliSense.

Reply:
(and they definitely don't match what we show in the docs: https://docs.microsoft.com/en-us/dotnet/api/system.numerics.quaternion?view=netcore-3.1)

Reply:
Why make it worse? At least before this change a reader didn't need to scroll. This change provides negative value IMO.

Reply:
I find it makes it better as these docs aren't relevant and it involves less scrolling to get to the bits you actually care about, the code.

It might be nice to either remove these docs (as they are massively out of date now anyways) or actually fix them to match what we have in the docs repo.

Reply:
@eerhardt, would you have a concern if I just removed the docs as part of this?

Reply:
If the docs are wrong or don't add value, then I wouldn't have any concern removing them.

However, in this case I think this summary is adding value, as most people (myself included) don't know what a Quaternion is off the top of their head.

Reply:
I changed the summary to `Represents a vector that is used to encode three-dimensional physical rotations.`, which matches the current docs and is shorter so the scrolling shouldn't be a concern.